### ISSUE TITLE
Collapsed toots too hard to distinguish from short, regular toots

### ISSUE THREAD

**Issue by Felthry** [glitch-soc/mastodon#353]
When you collapse a toot, if the "show image background" setting is turned off, it looks nearly identical to an ordinary, uncollapsed but very short one, certainly hard to tell apart at a glance. But if the "show image background" setting is on, the images can clutter the view, and can even be animated gifs which are generally unpleasant to see constantly playing in your home column.

I'd like to propose that either the default behavior be changed, or an option be added, that would allow the user to set a flat background color for collapsed toots. The behavior already exists, as it shows a flat black background if image backgrounds are on and the user in question doesn't have one set; there would just need to be an option to turn that on in the general case.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by beatrix-bitrot** [comment]
I agree that these are hard to distinguish. I'm not entirely sure about the proposed solution but definitely something can be done to make it more apparent that a toot has been collapsed...
